Events/app/src/main/java/com/dti/cornell/events/ProfileFragment.java

```diff
@@ -194,15 +194,15 @@ public void onClick(View view) {
 
 	public void configure(){
 		followingRecycler.setAdapter(new OrganizationAdapter(followingRecycler.getContext(),
-				Data.organizations().stream().filter((val)->OrganizationUtil.userIsFollowing(val.id)).collect(Collectors.toList()), false));
-		tagRecycler.setAdapter(new TagAdapter(tagRecycler.getContext(), ImmutableList.copyOf(TagUtil.getMostPopularTags(100)), false));
+				Data.organizations().stream().filter((val)->OrganizationUtil.userIsFollowing(val.id)).collect(Collectors.toList()), false, false));
```

Comment:
Not sure why we're using Java 8 streams here
Remember that they're only supported from API 24 ([see here](https://developer.android.com/studio/write/java8-support#supported_features))
(btw I see that that is the minimum version supported in `build.gradle` now, presumably to allow Java 8. I think that kind of decision is best made after looking at stats of Android users at Cornell. API 21 should be min in my opinion but I'm old and that might be outdated info.)